code-challenges.test.js

```diff
@@ -17,35 +17,166 @@
 
 const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = [
-  "chartreuse",
-  "indigo",
-  "periwinkle",
-  "ochre",
-  "aquamarine",
-  "saffron"
-]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// const colors2 = [
+//   "chartreuse",
+//   "indigo",
+//   "periwinkle",
+//   "ochre",
+//   "aquamarine",
+//   "saffron"
+// ]
+// // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+
+// describe("newColors", () => {
+//   it("create a function that takes an array and removes the first item and shuffles the remainder", () => {
+//     expect(newColors(colors1)).toEqual(["yellow", "blue", "pink", "green"])
+//     expect(newColors(colors2)).toEqual(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"])
+//   })
+// })
+
+//RED- Good failure
+// FAIL  ./code-challenges.test.js
+// newColors
+//   ✕ create a function that takes an array and removes the first item and shuffles the remainder
+
+// ● newColors › create a function that takes an array and removes the first item and shuffles the remainder
+
+//   ReferenceError: newColors is not defined
+
+//     30 | describe("newColors", () => {
+//     31 |   it("create a function that takes an array and removes the first item and shuffles the remainder", () => {
+//   > 32 |     expect(newColors(colors1)).toEqual(["yellow", "blue", "pink", "green"])
+//        |     ^
+//     33 |     expect(newColors(colors2)).toEqual(["saffron", "aquamarine", "periwinkle", "indigo", "ochre"])
+//     34 |   })
+//     35 | })
+
+//     at Object.expect (code-challenges.test.js:32:5)
+
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+// Snapshots:   0 total
 
 // b) Create the function that makes the test pass.
+//PseudoCode
+//Input: colors1
+//Output:["yellow", "blue", "pink", "green"] in varying orders
+//Process: create function called newColors with a parameter of any array, then create a new variable to store the new value, after removing the first value from the original array. Then, iterate through the new array with a for loop. Create a new variable to store the randomized shuffle, called newShuff followed by math.random and math.floor to shuffle the entirety of the new array [i]
```

Comment:
The thought process is there, and you are gathering good tools to use in your solution! I would continue to review Javascript fundamentals!